charts/super-agent-deployment/Chart.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v2
+name: super-agent-deployment
+description: A Helm chart to install New Relic Super agent on Kubernetes
+
+type: application
+version: 0.0.0-beta
+appVersion: TODO-FILL-THIS-WITH-CI
```

Comment:
Can you add a comment (in code or in PR) explaining this TODO a bit more?

Reply:
Done on [#6b31e5f](https://github.com/newrelic/helm-charts/pull/1155/commits/6b31e5fa129a97e2e585cf0117981ad3df4a4233)